modules/apps/notification/notification-rest-impl/src/main/java/com/liferay/notification/rest/internal/resource/v1_0/NotificationTemplateResourceImpl.java

```diff
@@ -343,6 +372,28 @@ private NotificationTemplate _toNotificationTemplate(
 				name = serviceBuilderNotificationTemplate.getName();
 				name_i18n = LocalizedMapUtil.getLanguageIdMap(
 					serviceBuilderNotificationTemplate.getNameMap());
+
+				if (GetterUtil.getBoolean(
+						PropsUtil.get("feature.flag.LPS-168220"))) {
+
+					setObjectDefinitionERC(
```

Comment:
@marcelabc I think the pattern is to leave the `set` methods  at the end, like in [here](https://github.com/liferay/liferay-portal/blob/8c67c06639def90c044ce511e6b31a1a8e38f3d5/modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java#L725-L742)